### ISSUE TITLE
Font doesn't work on my Ubuntu system

### ISSUE THREAD

**Issue by robtech21** [SandUhrGucker/Duco-Coin-Symbol-Icon-ttf-.h#1]
So I've been trying to get this to work on my "Le Potato" ARM SBC with Armbian Focal (Ubuntu 20.04) and even though I copied the `.ttf` file to the right directory and updated the cache it still doesn't work. Is there a way I can fix this?

**Comment by SandUhrGucker** [comment]
As described, the Font contains only one chatacter, the little "a". Did you try this to display? Send a sniplet of your code...

**Comment by robtech21** [comment]
> As described, the Font contains only one chatacter, the little "a". Did you try this to display? Send a sniplet of your code...

I'm just trying to get this to display on my OS (In terminal), do I need some other font to get it to show up?

**Comment by SandUhrGucker** [comment]
> I'm just trying to get this to display on my OS (In terminal), do I need some other font to get it to show up?

No, but you need to setup your console to support utf8/unicode. Then :
$ printf "\u1572"                                                              
ᕲ
$ echo -e "\u1572"                                                              
ᕲ

**Comment by robtech21** [comment]
> > I'm just trying to get this to display on my OS (In terminal), do I need some other font to get it to show up?
> 
> No, but you need to setup your console to support utf8/unicode. Then :
> $ printf "\u1572"
> ᕲ
> $ echo -e "\u1572"
> ᕲ

Ah, thanks. Another way I got it to display was by installing the `fonts-lg-aboriginal` package on Ubuntu, which displayed the character, but sadly it doesn't really seem to work with `figlet` through my testing so far. Someone will probably need to make a custom `figlet` font for it to work.
